provisioning_templates/provision/kickstart_default.erb

```diff
@@ -44,7 +44,19 @@ This template accepts the following parameters:
 install
 <% end -%>
 <% if host_param('kickstart_liveimg') %>
-liveimg --url=<%= host_param('kickstart_liveimg') %> <%= proxy_string %>
+<%
+img_name = host_param('kickstart_liveimg')
+liveimg_url = if host_param('kt_activation_keys')
+  repository_url(img_name, 'isos')
+else
+  if img_name.match(%r|^([\w\-\+]+)://|)
```

Comment:
This can easily raise `nil` exception.

Reply:
It can't, it is inside the true branch of `<% if host_param('kickstart_liveimg') %>` so it is guarranteed to be non-nil

Reply:
Oh fine missed that, thanks!